tests/filter-tests.js

```diff
@@ -438,4 +438,42 @@ describe("broccoli-asset-rev", function () {
       `,
     });
   });
+
+  it('can optionally use caching', async function() {
```

Comment:
Can you add a few more assertions around `output.changes()`?

* NOOP does not have changed files

```js
// NOOP
        await output.build();

        assert.deepEqual(output.changes(), {});
```

* Changed input only has the expected file changes:

```js
// UPDATE
        input.write({
          "a.herp": "AA", // change
          lib: null, // rmdir
        });
        await output.build();

        assert.deepEqual(
          output.read(),
          { "a.derp": "derp AA!" },
          'has expected updates'
        );
        assert.deepEqual(output.changes(), {
          "lib/c.derp": "unlink",
          "lib/b.derp": "unlink",
          "lib/": "rmdir",
          "a.derp": "change",
        });
```

Reply:
Done 👍